modules/microshift-nw-router-disable.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-the-router_{context}"]
+= Disabling the router
+
+In use cases such as industrial IoT spaces where {microshift-short} pods only need to connect to southbound operational systems and northbound cloud-data systems, inbound services are not needed. Use this procedure to disable the router in such egress-only use cases.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+//Q: Any other prereqs?
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameters in the `ingress.status` field to `Removed` as shown in the following example:
++
+.Example `config.yaml` ingress stanza
+[source,yaml]
+----
+apiServer:
+  advertiseAddress: 10.44.0.0/32
+#....
+ingress:
+  listenAddress:
+    - ""
```

Comment:
It is better specifying a host IP address or a host interface for the listenAddress, not specifying the empty value for it. Or else the cluster will crash after reloaded the microshift service.

```
example for the listenAddress:
ingress:
    listenAddress:
        - "10.1.235.168"
        - "ens4"
    ports:
    
or use the default listenAddress by removed it from the config.yaml:
ingress:
    ports:
```

Reply:
@ShudiLi My understanding is that disabling the router causes the service to skip the ingress fields because we are specifying egress-only. Is that not the case?

Reply:
@ShaunaDiaz Yes. I notice the configuration example has the listenAddress(I should added the comment in the above). For disabling the router, the config.yaml can be as below. 

```
ingress:
  status: Removed 
```

Reply:
@ShudiLi I think I captured your comments for `listenAddress`. (We do have a minimum requirement in the docs for YAML files, which is what I have here.)